buildconf.py

```diff
@@ -117,7 +118,9 @@ def clonePackage(cfg, package, server, gitPackage, branch):
             cmd = ["git", "clone", "-o", "autobuild", "-q", server+gitPackage, clonePath]
             if branch:
                 cmd += ["-b", branch]
-            execute.do(cmd, cfg)
+            out, err, r = execute.do(cmd, cfg)
+            print out
+            print err
```

Comment:
As @malter suggested:
```python
if err:
    print(err)
```

Reply:
@AlexanderFabisch at least it is usefull to have err information in any case.